net/dhtd/Makefile

```diff
@@ -0,0 +1,70 @@
+include $(TOPDIR)/rules.mk
+
+
+PKG_NAME:=dhtd
+PKG_VERSION:=0.1.0
+PKG_RELEASE:=1
+
+PKG_LICENSE:=MIT
+
+PKG_SOURCE_URL:=https://codeload.github.com/mwarning/dhtd/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE:=dhtd-$(PKG_VERSION).tar.gz
+PKG_HASH:=13c3a2c5ce5d35d5aa1b156b13dcc489456ee61efc3020dff5e34d18843666ef
+PKG_BUILD_DIR:=$(BUILD_DIR)/dhtd-$(PKG_VERSION)
+
+
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/dhtd
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=DHT Daemon
+	SUBMENU:=IP Addresses and Names
+	URL:=https://github.com/mwarning/dhtd
+	MENU:=1
+	MAINTAINER:=Moritz Warning <moritzwarning@web.de>
+endef
+
+define Package/dhtd/description
+	Standalone BitTorrent DHT daemon. Lookup and announce hash identifiers via command line interface.
+endef
+
+define Package/dhtd/config
+	source "$(SOURCE)/Config.in"
+endef
+
+FEATURES = ""
+
+ifeq ($(CONFIG_DHTD_ENABLE_CLI),y)
+FEATURES += cli
+endif
+
+ifeq ($(CONFIG_DHTD_ENABLE_LPD),y)
+FEATURES += lpd
+endif
+
+ifeq ($(CONFIG_DHTD_ENABLE_DEBUG),y)
+FEATURES += debug
+endif
+
+MAKE_FLAGS += FEATURES="$(FEATURES)"
+
+# Make binary smaller
+TARGET_CFLAGS += -ffunction-sections -fdata-sections
+TARGET_LDFLAGS += -Wl,--gc-sections
+
+define Package/dhtd/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/build/dhtd $(1)/usr/bin/
+ifeq ($(CONFIG_DHTD_ENABLE_CLI),y)
```

Comment:
It is more common to move this to a own package `dhtd-utils` on do not enable/disable it via Config.in.

Reply:
OK, I have removed the configuration options.